### PR TITLE
test: run APM Server managed

### PR DESCRIPTION
**Pull request by kuisathaverat**
## What does this PR do?

it allows to test the APM Server launched in managed way.

## Why is it important?

We have to test the new way to launch the APM Server using the Elastic Agent.

## Related issues
related to https://github.com/elastic/apm-integration-testing/pull/1016

### CODE REVIEW

File: .ci/Jenkinsfile [hunk 1]
```diff
@@ -34,7 +34,7 @@ pipeline {
     string(name: 'ELASTIC_STACK_VERSION', defaultValue: "8.0.0", description: "Elastic Stack Git branch/tag to use")
     string(name: 'BUILD_OPTS', defaultValue: "", description: "Additional build options to pass to compose.py")
     string(name: 'SLACK_CHANNEL', defaultValue: 'observablt-bots', description: 'The Slack channel where errors will be posted')
-    booleanParam(name: 'Run_As_Master_Branch', defaultValue: false, description: 'Allow to run any steps on a PR, some steps normally only run on master branch.')
+    booleanParam(name: 'Run_As_Master_Branch', defaultValue: true, description: 'Allow to run any steps on a PR, some steps normally only run on master branch.')
```
Comment (v1v) [inline .ci/Jenkinsfile:37]: what's the reason?

Comment (kuisathaverat) [reply .ci/Jenkinsfile:37]: to test this Draft PR

**Comment by kuisathaverat** [comment]
we have to wait for a way to inject the APM Server in the Elastic Agent image from code or use a binary build after every merge.

File: .ci/Jenkinsfile (hunk 1 above)
Comment (kuisathaverat) [reply .ci/Jenkinsfile:38]:
```suggestion
    booleanParam(name: 'Run_As_Master_Branch', defaultValue: false, description: 'Allow to run any steps on a PR, some steps normally only run on master branch.')
```

**Review by cachedout — APPROVED** [review]

**Pull request merged**